app/helpers/application_helper.rb

```diff
@@ -148,17 +148,14 @@ def locked_icon(condition, hovertext)
     ('<span class="glyphicon glyphicon-lock" title="%s"/>' % hovertext).html_safe if condition
   end
 
-  def searchable?
+  def searchable?(filter_columns = false)
     return false if !User.current || @welcome || @missing_permissions
```

Comment:
I guess Shim meant extract this line in a method that can be used in `searchable?` and `filter_columns?`, but I might be wrong as well...

It just seems weird that `searchable?` is responsible for columns filter...

Reply:
> I guess Shim meant extract this line in a method that can be used in searchable? and filter_columns?, but I might be wrong as well...

You are correct, that's what I meant.

Reply:
Sorry about that, updated.